plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -166,6 +171,11 @@ public MachineProviderImpl(DockerConnector docker,
         this.doForcePullOnBuild = doForcePullOnBuild;
         this.privilegedMode = privilegedMode;
         this.snapshotUseRegistry = snapshotUseRegistry;
+        if (Strings.isNullOrEmpty(cheUserId)) {
+            this.cheUserId = System.getenv("CHE_USER_ID") == null ? "" : System.getenv("CHE_USER_ID");
```

Comment:
I don't mean binding  CHE_PROPERTY_che_docker_user__id. I mean that now we use properties. Then to overwrite it we add env variable into che.env file.